templates/rhpam73-authoring-ha.yaml

```diff
@@ -1269,21 +1299,130 @@ objects:
+      spec:
+        serviceAccountName: ${APPLICATION_NAME}-amq-scaledown-controller-sa
+        containers:
+        - args:
+          - --localOnly
+          - --v
+          - "3"
+          image: ${AMQ_SCALEDOWN_CONTROLLER_IMAGE_STREAM_NAME}
+          imagePullPolicy: IfNotPresent
+          name: ${APPLICATION_NAME}-amq
+        terminationGracePeriodSeconds: 30
+    triggers:
+    - imageChangeParams:
+        automatic: true
+        containerNames:
+        - ${APPLICATION_NAME}-amq
+        from:
+          kind: ImageStreamTag
+          namespace: "${AMQ_SCALEDOWN_CONTROLLER_IMAGE_STREAM_NAMESPACE}"
+          name: "${AMQ_SCALEDOWN_CONTROLLER_IMAGE_STREAM_NAME}:${AMQ_SCALEDOWN_CONTROLLER_IMAGE_STREAM_TAG}"
+      type: ImageChange
+    - type: ConfigChange
+- kind: StatefulSet
+  apiVersion: apps/v1
+  metadata:
+    annotations:
+      alpha.image.policy.openshift.io/resolve-names: '*'
+      statefulsets.kubernetes.io/drainer-pod-template: |
+        metadata:
+          labels:
+            app: "${APPLICATION_NAME}-amq-drainer"
+          name: "${APPLICATION_NAME}-amq-drainer-pod"
+        spec:
+          serviceAccount: "${APPLICATION_NAME}-amq-sa"
+          serviceAccountName: "${APPLICATION_NAME}-amq-sa"
+          terminationGracePeriodSeconds: 5
+          containers:
+          - env:
+            - name: AMQ_EXTRA_ARGS
+              value: "--no-autotune"
+            - name: AMQ_USER
+              value: "${APPFORMER_JMS_BROKER_USER}"
+            - name: AMQ_PASSWORD
+              value: "${APPFORMER_JMS_BROKER_PASSWORD}"
+            - name: AMQ_ROLE
+              value: "${AMQ_ROLE}"
+            - name: AMQ_NAME
+              value: "${AMQ_NAME}"
+            - name: AMQ_TRANSPORTS
+              value: openwire
+            - name: AMQ_QUEUES
+              value: ''
+            - name: AMQ_GLOBAL_MAX_SIZE
+              value: "${AMQ_GLOBAL_MAX_SIZE}"
+            - name: AMQ_ALLOW_ANONYMOUS
+              value: "${AMQ_ALLOW_ANONYMOUS}"
+            - name: AMQ_DATA_DIR
+              value: "/opt/amq/data"
+            - name: AMQ_DATA_DIR_LOGGING
+              value: 'true'
+            - name: AMQ_CLUSTERED
+              value: 'true'
+            - name: AMQ_REPLICAS
+              value: '2'
+            - name: AMQ_CLUSTER_USER
+              value: "${APPFORMER_JMS_BROKER_USER}"
+            - name: AMQ_CLUSTER_PASSWORD
+              value: "${APPFORMER_JMS_BROKER_PASSWORD}"
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: OPENSHIFT_DNS_PING_SERVICE_PORT
+              value: '8888'
+            image: "${AMQ_BROKER_IMAGE}"
+            name: "${APPLICATION_NAME}-amq"
+            command:
+            - "/bin/sh"
+            - "-c"
+            - echo "Starting the drainer" ; /opt/amq/bin/drain.sh; echo "Drain completed!
+              Exit code $?"
```

Comment:
accidental new line?

Reply:
This was originally a JSON file and I converted using an online tool. This new line didn't break the template, but if you think it is better keep in the same line I can do it.

Reply:
please do it.